android/app/build.gradle

```diff
@@ -161,6 +161,10 @@ android {
 
         versionCode 11
         versionName "1.0.8"
+
+        //Fix problem in bt where .env.bt is not recognized
+        resValue "string", "build_config_package", "org.pathcheck.covidsafepaths"
```

Comment:
Oh wow, this is great!!! just tested it and it definitely fixes the problem. If you have the time would you mind educating me on what the issue was? Also, we had an open issue on the react-native-config package https://github.com/luggit/react-native-config/issues/467 and this fixes that question we opened if you are ok with linking this PR to the issue we can close it right away. I'll wait a bit for some other folks to check it out but I'm more than happy with merging this. 🎉🎉🎉🎉🎉

Reply:
I set a breakpoint in the react-native-config file ReactNativeConfigModule.java.  I saw that it was trying to get the Config constants from org.pathcheck.covidsafepathsBt.BuildConfig, which didn't exist.  So I tried changing the path to org.pathcheck.covidsafepaths.BuildConfig, and that worked.  Then I remembered this from the react-native-config README:
````
In `android/app/build.gradle`, if you use `applicationIdSuffix` or `applicationId` that is different from the package name indicated in `AndroidManifest.xml` in `<manifest package="...">` tag, for example, to support different build variants:
Add this in `android/app/build.gradle`

```
defaultConfig {
    ...
    resValue "string", "build_config_package", "YOUR_PACKAGE_NAME_IN_ANDROIDMANIFEST.XML"
}
````
I didn't see that the condition that was stated applied, but I decided to try it anyway, and I was lucky